example/cargo-doc-live/flake.nix

```
@@ -0,0 +1,44 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    systems.url = "github:nix-systems/default";
+    process-compose-flake.url = "github:Platonic-Systems/process-compose-flake";
+    services-flake.url = "github:juspay/services-flake";
+  };
+  outputs = inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = import inputs.systems;
+      imports = [
+        inputs.process-compose-flake.flakeModule
+      ];
+      perSystem = { self', pkgs, lib, ... }: {
+        # `process-compose.foo` will add a flake package output called "foo".
+        # Therefore, this will add a default package that you can build using
+        # `nix build` and run using `nix run`.
+        process-compose."default" = { ... }:
```

```suggestion
        process-compose."cargo-doc-live" = { ... }:
```

(The default package is usually the Rust package itself)